### ISSUE TITLE
place element off screen such that it is not visible but can be read by screen reader

### ISSUE THREAD

**Issue by DeepakSap14** [SAP/fundamental-styles#2477]
### Component name
> offScreen

### Detailed description
> Describe the request
For aria-live to read any change in element, we need to place element but it should not be visible to user.
same time, it can not be made `hidden` or `display: none`, as screen reader will not be able to pick this element.

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
NA

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
Angular

### Use case
> Describe when to use (and when not to use)
For aria-live to read any change in element, we need to place element but it should not be visible to user.
same time, it can not be made `hidden` or `display: none`, as screen reader will not be able to pick this element.

### Visual style
> Describe specific needs related to color, typography, iconography, borders, space, size, and other visual properties.
NA
> Describe how to arrange multiple instances and other alignment specifics.
NA
> How should this respond to common contexts, i.e., touch, RTL, small screens.
NA
> Describe interactive states, e.g., disabled, expanded, pressed.
NA

### Expected behavior
> How should this handle events, outcomes, and transitions
NA
> What [states should be available](https://medium.com/swlh/the-nine-states-of-design-5bfe9b3d6d85), i.e., nothing, loading, none, one, some, too many, incorrect, correct, done.
NA

### Editorial
> Advise on labels, tone, length, punctuation, truncation.
NA

## Additional information

* Screenshots or code
* Notes
NA
